Feature/cache policy (#47)
infinum/ios-prince-of-versions: README.md

#### Getting all data

  ```swift
    let princeOfVersionsURL = URL(string: "https://pastebin.com/raw/0MfYmWGu")!

    PrinceOfVersions.checkForUpdates(
        from: princeOfVersionsURL,
        cachePolicy: .reloadIgnoringLocalAndRemoteCacheData,
        completion: { [unowned self] response in
            switch response.result {
            case .success(let updateResultData):
                print("Update version: \(updateResultData.updateVersion)")
                print("Installed version: \(updateResultData.updateInfo.installedVersion)")
                print("Update status: \(updateResultData.updateStatus)")
            case .failure:
                // Handle error
                break
            }
        }
    )
  ```
  
#### Request cache policy

When checking for updates, be it for AppStore or other sources, you can provide a *cachePolicy* parameter. The *cachePolicy* parameter determines which request cache policy will be used for network requests.

> The initial value for the default `URLSession` configuration uses **.useProtocolCachePolicy** which shouldn't be the default behavior for Prince of Versions, we want to refresh the data from the server every time. That's why we defaulted the value to **.reloadIgnoringLocalAndRemoteCacheData**. If you want to change the default behavior, you can always provide which cache policy you want to use, as seen in the example above :)

Reviewer: ```suggestion
The initial value for the default `URLSession` configuration uses **.useProtocolCachePolicy**, which shouldn't be the default behavior for Prince of Versions; we want to refresh the data from the server every time. That's why we defaulted the value to **.reloadIgnoringLocalAndRemoteCacheData**. If you want to change the default behavior, you can always provide which cache policy you want to use, as seen in the example above.
```

It's a bit more professional without the smiley 😄

Reviewer: If you remove the call site setting as I mentioned in the comment above, then we should also update the last sentence to be sth like: 

```suggestion
The initial value for the default `URLSession` configuration uses **.useProtocolCachePolicy** which shouldn't be the default behavior for Prince of Versions, we want to refresh the data from the server every time. That's why we defaulted the value to **.reloadIgnoringLocalAndRemoteCacheData**. If you want to change the default behavior, you can always provide a different cache policy that suits your needs.
```
